apps/bmg-news-app/src/app/screens/MainScreen.tsx

```diff
@@ -1,17 +1,19 @@
-import { Text, View } from 'react-native';
+import { Text, View, SafeAreaView } from 'react-native';
 
 import { MyComponent } from '@bmg-react-native-nx/core';
+import { ViewsFormatter } from '@bmg-react-native-nx/core';

```

Comment:
როდესაც რამდნეიმე კომპონენტი, ერთი და იგივე `path`-დან მოდის, ჯობია ერთ იმპორტში გვქონდეს

```suggestion
import { MyComponent, ViewsFormatter  } from '@bmg-react-native-nx/core';

```